src/fhir/PlanDefinition.json

```diff
@@ -247,6 +250,27 @@
         }
       ]
     },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "AutumnMsg",
+      "name": "Autumn Message",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "event": [
+          "2023-09-25T15:33:00.000Z"
+        ]
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "{name}, welcome to a season of change! Though change can be scary, it can be very beautiful. What positive changes have you seen this season? -OPC care team"
```

Comment:
@mcjustin Perhaps we want to remove `-OPC care team` here since this is the default PlanDefinition?

Reply:
done, thanks